README.md

```diff
@@ -474,6 +474,14 @@ $sugar_config['verify_client_ip'] = false;
 ```
 Tweak the above settings based on your specific needs.
 
+### PHPMyAdmin
+You can run PHPMyAdmin in a container to get access to the database tables.
```

Comment:
Thanks @kwalker-sugarcrm for the contribution!
Could you please add a quick note about how to get the network name? As it will be dependent on the stack you have in use.
Also, if we go by everything else on the documentation, the url should probably be http://docker.local:8080

Thanks!

Reply:
added info on getting the network name.

The docker command is forwarding port 80 to 8080 on localhost, so docker.local is not correct, at least on my machine.